component/pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>org.wso2.extension.siddhi.map.wso2event</groupId>
         <artifactId>siddhi-map-wso2event-parent</artifactId>
-        <version>5.0.1</version>
+        <version>5.0.1-SNAPSHOT</version>
```

Comment:
it should be 5.0.2-snapshot I believe. Check last released

Reply:
Last release is 5.0.0

Reply:
Seems [commit](https://github.com/wso2-extensions/siddhi-map-wso2event/commit/29ef82b0c479080f15b78249cd27d669847be39b) updated the version